### DISCUSSION TITLE
Rockchip Radxa 5C Lite

### DISCUSSION THREAD

**Discussion by peltho** [kwankiu/archlinux-installer-rock5#69] [Q&A] [answered]
Thank you for your work!
I'm wondering if I can use it for Radxa 5C Lite machines since they are RK3582 and not RK3588 like 5C.

**Comment by kwankiu** [comment, marked as answer]
> Thank you for your work! I'm wondering if I can use it for Radxa 5C Lite machines since they are RK3582 and not RK3588 like 5C.

Yes it automatically checks and enables working IP cores just like armbian's images, if you are lucky it will work exactly like a RK3588S

**Reply by peltho** [reply]
Thanks for your answer! If someone can confirm it would be great! 🙏🏻

**Reply by kwankiu** [reply]
I dont have a Rock 5C Lite, but I have a CM5 Lite which works well, it is also using the RK3582

**Reply by peltho** [reply]
That's good to know
